src/base/utils/string.cpp

```diff
@@ -177,8 +177,12 @@ QString Utils::String::fromStdString(const std::string &str)
 
 std::string Utils::String::toStdString(const QString &str)
 {
+#ifdef QBT_USES_QT5
```

Comment:
@evsh be sure to include this in your #6266.

@ all : I am planning to do a final v3.3.11 release this weekend with some very specific commits and the whole current master. After that, we drop Qt4 in master and v3_3_x enters maintenance mode.
maintenance mode: Fix broken stuff only(if any), until I decide to do the next major release (probably named 3.4.0).